### PR TITLE
Minor additions to UI, which makes customizing easier

### PR DESCRIPTION
**Pull request by necccc**
Added some really minor properties and unobtrusive additions to the UI elements of the menu and prompt components, which add some UX goodness, and make them more customizable from CSS for example.

Such as:
- every menuitem has it's command in a data property
- add a submit and a cancel action to the prompt, additional to the closing X
- the prompt contains a title, from the action command name

These are all hidden by default in the CSS, so existing demos and usecases are not supposed to be affected.

### CODE REVIEW

File: src/menu/menu.js [hunk 1]
```diff
@@ -219,7 +219,11 @@ function renderDropdownItems(items, pm) {
   let rendered = []
   for (let i = 0; i < items.length; i++) {
     let inner = items[i].render(pm)
-    if (inner) rendered.push(elt("div", {class: prefix + "-dropdown-item"}, inner))
+    if (inner) {
+      let element = elt("div", {class: prefix + "-dropdown-item"}, inner)
+      if (items[i].command_) element.setAttribute("data-cmd", items[i].command_.name)
```
Comment (marijnh) [inline src/menu/menu.js]: `.command_` is allowed to be a string, in which case you shouldn't dereference its `name` property.

Comment (marijnh) [inline src/menu/menu.js]: Also wouldn't it make more sense for `MenuCommand` to be doing the attribute attaching (to the inner node)?

File: src/ui/prompt.js [hunk 2]
```diff
@@ -31,9 +31,12 @@ export class ParamPrompt {
         AssertionError.raise("Unsupported parameter type: " + param.type)
       return this.paramTypes[param.type].render.call(this.pm, param, this.defaultValue(param))
     })
+    let promtTitle = elt("h5", {}, (command.spec && command.spec.label) ? command.spec.label : '')
```
Comment (marijnh) [inline src/ui/prompt.js]: Spelling mistake ('prompt')

File: src/ui/prompt.js [hunk 3]
```diff
@@ -31,9 +31,12 @@ export class ParamPrompt {
...
+    let submitButton = elt("button", {type: "submit", class: "ProseMirror-prompt-submit"}, "Ok")
+    let cancelLink = elt("a", {class: "ProseMirror-prompt-cancel"}, "Cancel")
```
Comment (marijnh) [inline src/ui/prompt.js]: There's already a close icon added by `openPrompt`.

Comment (necccc) [inline src/ui/prompt.js]: That is true, although it's nice to have them both from an UX perspective (the ux team, with we're integrating prosemirror in a usecase, suggested this - along with the prompt title)

File: src/menu/menu.js (hunk 1 above)
Comment (necccc) [inline src/menu/menu.js]: Originally I did that, but during styling traversing within sibling nodes was much much easier

**Comment by marijnh** [comment]
I've pushed your changes as fe3db74b1b3 and followed up with 84f8dd6. I made the title visible by default, and changed the structure of the submit/cancel buttons a bit (making them both buttons, wrapped in an additional div). I've also moved the `data-cmd` attribute to be `data-command` on the icon itself. Having wrapping renderers (of which people can write their own variants) check every time whether the thing rendered is a command is just not a good idea.

**Pull request closed without merging**